data/gui/window/hotkey_transliteration.cfg

```diff
@@ -0,0 +1,197 @@
+											grow_factor = 1
+											horizontal_grow = true
+											[grid]
+												[row]
+													[column]
+														grow_factor = 1
+														horizontal_alignment = "left"
+														border = "top,bottom,right"
+														border_size = 5
+
+														[label]
+															id = "name"
+															definition = "gold_small"
+														[/label]
+													[/column]
+												[/row]
+
+												[row]
+													[column]
+														grow_factor = 1
+														horizontal_alignment = "left"
+														border = "top,bottom,right"
+														border_size = 5
+
+														[label]
+															id = "example"
+															definition = "small"
+														[/label]
+													[/column]
+												[/row]
+											[/grid]
+										[/column]
+									[/row]
+								[/grid]
+							[/toggle_panel]
+						[/column]
+					[/row]
+				[/list_definition]
+			[/listbox]
+		[/column]
+	[/row]
+#enddef
+
+[window]
+	id = "hotkey_transliteration"
+	description = "Supporting hot-keys over multiple keyboard layouts."
+
+	[resolution]
+		definition = "default"
+
+		click_dismiss = true
+		maximum_width = "(max(600, screen_width * 4 / 5))"
+
+		[tooltip]
+			id = "tooltip"
+		[/tooltip]
+
+		[helptip]
+			id = "tooltip"
+		[/helptip]
+
+		[grid]
+
+			[row]
+				grow_factor = 0
+
+				[column]
+					grow_factor = 1
+
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+					[label]
+						id = "title"
+						definition = "title"
+
+						label = _ "Hot-keys In Multiple Keyboard Layouts"
+					[/label]
+
+				[/column]
+
+			[/row]
+
+			[row]
+				grow_factor = 0
+
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+					[label]
+						definition = "default"
+
+						# po: I'm using Greek for the examples because (looking at Wikipedia) there seems to
+						# be consistency in which Greek letters match to which Latin letters. Cyrillic on
+						# the other hand seems to have as many keyboard variants as Latin does (Qwerty,
+						# Qwertz, Azerty, etc),
+						label = _ "This makes the hot-keys work even when switching between keyboard layouts.
+
+For example, if your two layouts have Latin "b" and Greek "β" on the same key, enabling “Greek letters to Latin letters” will add a fallback so that pressing ctrl+β will trigger the hot-key for ctrl+b."
+						wrap = true
+					[/label]
+
+				[/column]
+
+			[/row]
+
+			[row]
+				grow_factor = 1
+
+				[column]
+					horizontal_grow = true
+					vertical_alignment = "top"
+
+					[grid]
+						{_GUI_HOTKEY_TRANSLITERATION_GRID}
+					[/grid]
+				[/column]
+
+			[/row]
+
+			[row]
+				grow_factor = 0
+
+				[column]
+					horizontal_alignment = "right"
+
+					[grid]
+
+						[row]
+							grow_factor = 0
+
+							[column]
+								border = "all"
+								border_size = 5
+								horizontal_alignment = "right"
+
+								[button]
+									id = "ok"
```

Comment:
Usually the "Close" button uses the id "cancel", which allows Esc to close the dialog. Since it is using ok, it gets closed when Enter is pressed. Is that intended?